independent-projects/tools/common/src/test/java/io/quarkus/test/platform/descriptor/loader/QuarkusTestPlatformDescriptorLoader.java

```diff
@@ -139,7 +140,12 @@ public String getTemplate(String name) {
             @Override
             public <T> T loadResource(String name, ResourceInputStreamConsumer<T> consumer) throws IOException {
                 return loadStaticResource(name, consumer);
-            }};
+            }
+
```

Comment:
Formatting issue.

Reply:
See above comments - independent projects .java file were NOT formatted even before this PR.

Reply:
file now using quarkus format.

Reply:
file now using quarkus format.